src/main/resources/feedbackQuestionRubricEditFormTableOptionsFragment.html

```diff
@@ -0,0 +1,16 @@
+<td class="align-center rubricCol-${questionIndex}-${col}" data-col="${col}">
+    <div class="btn-group">
+        <button type="button" class="btn btn-default" id="rubricMoveChoiceLink-${questionIndex}-${col}-l" onclick="swapRubricCol(${col}, ${questionIndex}, true)"
+            data-toggle="tooltip" data-placement="top" title="Move column left">
+            <span class="glyphicon glyphicon-arrow-left"></span>
+        </button>
+        <button type="button" class="btn btn-default" id="rubricRemoveChoiceLink-${questionIndex}-${col}" onclick="removeRubricCol(${col}, ${questionIndex})"
```

Comment:
Missed this earlier: any `id` or `class` should also be in lowercase, with hyphens between words.